llvm/lib/SYCLPostLink/ComputeModuleRuntimeInfo.cpp

```diff
@@ -106,13 +107,20 @@ traverseCGToFindSPIRKernels(const Function *StartingFunction) {
   return {std::move(KernelNames)};
 }
 
-static std::vector<StringRef> getKernelNamesUsingAssert(const Module &M) {
-  auto *DevicelibAssertFailFunction = M.getFunction("__devicelib_assert_fail");
-  if (!DevicelibAssertFailFunction)
+static std::vector<StringRef>
+getKernelNamesUsingSpecialFunctions(const Module &M,
+                                    std::vector<StringRef> &FNames) {
```

Comment:
```suggestion
                                    const std::vector<StringRef> &FNames) {
```

I think this can be const.